src/pages/ProductList/ProductList.js

```diff
@@ -1,7 +1,45 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import './ProductList.scss';
+import ProductListHeader from './ProductListHeader/ProductListHeader';
+import ProductListBanner from './ProductListBanner/ProductListBanner';
+import ProductListTitle from './ProductListTitle/ProductListTitle';
+import Product from './Product/Product';
+function ProductList() {
+  const [categories, setCategories] = useState([]);
 
-const ProductList = () => {
-  return <div>프로덕트리스트페이지입니다.</div>;
-};
+  useEffect(() => {
+    fetch('/data/categoriesData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setCategories(data);
+      });
+  }, []);
+
+  return (
+    <>
+      <nav>
+        nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바
+        자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리
+        nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리
+      </nav>
+
+      <ProductListHeader />
+
+      <ProductListBanner />
+
+      <main className="main">
+        <ProductListTitle />
+
+        {categories.map(function (category) {
+          return (
+            <Product category={category} i={category.id} key={category.id} />
+          );
+        })}
+      </main>
+    </>
```

Comment:
- ui 부분은 따로 개행 처리 하지 않습니다!
```suggestion
    <>
      <nav>
        nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바
        자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리
        nav바 자리 nav바 자리 nav바 자리 nav바 자리 nav바 자리
      </nav>
      <ProductListHeader />
      <ProductListBanner />
      <main className="main">
        <ProductListTitle />
        {categories.map(function (category) {
          return (
            <Product category={category} i={category.id} key={category.id} />
          );
        })}
      </main>
    </>
```

Reply:
넵!